packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -147,7 +147,7 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return (
       <div className={ textFieldClassName }>
         { label && <Label htmlFor={ this._id }>{ label }</Label> }
-        <div className={ css(styles.fieldGroup) }>
+        <div className={ css(styles.fieldGroup, isFocused && styles.fieldGroupIsFocused) }>
```

Comment:
The object map notation is kind of nice, but really it is weird that we have 2 different ways to do the same thing and it would be nice to be consistent. If we can move away from the object maps, that opens the door to using the new helper. I will need to call it something other than `css` to differentiate the behavior.